### ISSUE TITLE
grpcio-tools 1.27.0 released before grpcio 1.27.0, breaking install of the former

### ISSUE THREAD

**Issue by MichaelSaah** [grpc/grpc#21922]
## minimum reproduction:

```
$ docker run -it python:3.7.6-slim-stretch /bin/bash            
root@21672e6c27db:/# pip install grpcio-tools
Collecting grpcio-tools
  Downloading grpcio_tools-1.27.0-cp37-cp37m-manylinux2010_x86_64.whl (2.4 MB)
     |████████████████████████████████| 2.4 MB 2.0 MB/s
Collecting protobuf>=3.5.0.post1
  Downloading protobuf-3.11.3-cp37-cp37m-manylinux1_x86_64.whl (1.3 MB)
     |████████████████████████████████| 1.3 MB 11.3 MB/s
ERROR: Could not find a version that satisfies the requirement grpcio>=1.27.0 (from grpcio-tools) (from versions: 0.4.0a0, 0.4.0a1, 0.4.0a2, 0.4.0a3, 0.4.0a4, 0.4.0a5, 0.4.0a6, 0.4.0a7, 0.4.0a8, 0.4.0a13, 0.4.0a14, 0.5.0a0, 0.5.0a1, 0.5.0a2, 0.9.0a0, 0.9.0a1, 0.10.0a0, 0.11.0b0, 0.11.0b1, 0.12.0b0, 0.13.0, 0.13.1rc1, 0.13.1, 0.14.0rc1, 0.14.0, 0.15.0, 1.0.0rc1, 1.0.0rc2, 1.0.0, 1.0.1rc1, 1.0.1, 1.0.2rc0, 1.0.2, 1.0.3, 1.0.4, 1.1.0rc1, 1.1.0, 1.1.3, 1.2.0rc1, 1.2.0rc2, 1.2.0, 1.2.1, 1.3.0, 1.3.5, 1.4.0rc1, 1.4.0, 1.6.0rc1, 1.6.0, 1.6.3, 1.7.0rc1, 1.7.0, 1.7.3, 1.8.0rc1, 1.8.0rc2, 1.8.0rc3, 1.8.1, 1.8.2, 1.8.3, 1.8.4, 1.8.6, 1.9.0rc1, 1.9.0rc2, 1.9.0rc3, 1.9.0, 1.9.1, 1.10.0rc1, 1.10.0rc2, 1.10.0, 1.10.1rc1, 1.10.1rc2, 1.10.1, 1.11.0rc1, 1.11.0rc2, 1.11.0, 1.11.1rc1, 1.11.1, 1.12.0rc1, 1.12.0, 1.12.1, 1.13.0rc1, 1.13.0rc2, 1.13.0rc3, 1.13.0, 1.14.0rc1, 1.14.0rc2, 1.14.0, 1.14.1, 1.14.2rc1, 1.14.2, 1.15.0rc1, 1.15.0, 1.16.0rc1, 1.16.0, 1.16.1rc1, 1.16.1, 1.17.0rc1, 1.17.0, 1.17.1rc1, 1.17.1, 1.18.0rc1, 1.18.0, 1.19.0rc1, 1.19.0, 1.20.0rc1, 1.20.0rc2, 1.20.0rc3, 1.20.0, 1.20.1, 1.21.0rc1, 1.21.1rc1, 1.21.1, 1.22.0rc1, 1.22.0, 1.22.1, 1.23.0rc1, 1.23.0, 1.23.1, 1.24.0rc1, 1.24.0, 1.24.1, 1.24.3, 1.25.0rc1, 1.25.0, 1.26.0rc1, 1.26.0, 1.27.0rc1, 1.27.0rc2)
ERROR: No matching distribution found for grpcio>=1.27.0 (from grpcio-tools)
root@21672e6c27db:/#
```

`grpcio-tools` is looking for `grpcio` of the same version: https://github.com/grpc/grpc/blob/v1.27.0/tools/distrib/python/grpcio_tools/setup.py#L223

which as of now is not released yet: https://pypi.org/project/grpcio/#history

## workaround
pin to 1.26.0 for both deps for now

**Comment by adamf** [comment]
This is breaking builds for us as well. You need to pin _all_ grpcio packages (-tools, -health-checks, etc) to 1.26.0 or older.

**Comment by gnossen** [comment]
The 1.27.0 release of `grpcio` was deleted due to user-reported problems. `grpcio-tools` 1.27.0 has now also been deleted. This should resolve your issue. Thank you for the report!

**Comment by MichaelSaah** [comment]
thanks!

**Comment by michaelbeaumont** [comment]
@gnossen  It's generally accepted bad practice to delete released versions, why does `grpc` do this and shouldn't there be a place where events like version deletions are announced?

**Comment by gnossen** [comment]
@michaelbeaumont I empathize and agree to an extent. Deleting an artifact will affect everyone who has updated their project to explicitly pull in the new version. However, in cases where a release is known to be causing acute breakage across the user base, it will fix the much greater proportion who are pulling in the package using a version range.

In the 1.27.0 release, we dropped support for manylinux1. Unfortunately, we quickly received notice that a staggering number of users were broken by this change. Based on our knowledge of the extent of the breakage and a search of Github, we determined that this particular deletion would help many more users than it would hurt. So, less than 24 hours after it was initially published, we deleted the 1.27.0 release.

I agree that this could have been announced more clearly. While I sincerely hope we do not have to delete any releases in the future, we will announce any such actions on [the grpc-io mailing list](https://groups.google.com/g/grpc-io) going forward.

A 1.27.1 release with manylinux1 artifacts re-added will be up very soon.
